.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: check out code
+      uses: actions/checkout@v2
+
+    - name: setup go env
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: install tools
+      run: make install-ci
+
+    - name: run unit tests
+      run: make test-ci
+
+    - name: prepare coverage file
+      run: mv cover.out coverage.txt
+
+    - name: run codecov bash uploader
```

Comment:
Would it make sense to use their Github Action? 

https://github.com/codecov/codecov-action

Reply:
Sure.